src/main/resources/lib/menu/graphql.js

```diff
@@ -0,0 +1,107 @@
+var libs = {
+  graphql: require("/lib/graphql"),
```

Comment:
`lib-graphql` should be declared as a dependency in `build.gradle`. here the code "assumes" that `lib-graphql` is used by the app. I'm surprised it works at all.

And, if we have `lib-graphql` as a dependency, we'll have to up the `xpVersion` (currently set to 7.0.0) to support the version of `lib-graphql` which introduced `schemaGenerator`.

Reply:
The file */lib/menu.js* has not been changed at all, and would still work as before.

If you are importing from `/lib/menu/graphql.js` to get the schemas, you are explicitly doing it to use with GraphQL, and I had expected the application to already have it installed.